source/extensions/tracers/opentelemetry/samplers/cel/cel_sampler.h

```diff
@@ -37,7 +38,6 @@ class CELSampler : public Sampler, Logger::Loggable<Logger::Id::tracing> {
 private:
   const ::Envoy::LocalInfo::LocalInfo& local_info_;
   Extensions::Filters::Common::Expr::BuilderInstanceSharedPtr builder_;
-  const google::api::expr::v1alpha1::Expr parsed_expr_;
```

Comment:
Another instance where you deleted the storage for the expression.

Reply:
I wonder why we don't detect this in Asan - maybe the test lacks an expression with strings? Or the parent expression is not on stack?

Reply:
Not sure why ASAN is not able to catch these. Maybe because test execution pattern is different?